src/main/java/com/softserve/teamproject/validation/impl/StudentValidatorImpl.java

```diff
@@ -0,0 +1,59 @@
+package com.softserve.teamproject.validation.impl;
+
+import com.softserve.teamproject.entity.Group;
+import com.softserve.teamproject.entity.Student;
+import com.softserve.teamproject.entity.User;
+import com.softserve.teamproject.repository.StudentRepository;
+import com.softserve.teamproject.repository.UserRepository;
+import com.softserve.teamproject.validation.StudentValidator;
+import java.lang.reflect.Field;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.access.AccessDeniedException;
+import org.springframework.stereotype.Component;
+
+@Component
+public class StudentValidatorImpl implements StudentValidator {
+
+  private StudentRepository studentRepository;
+  private UserRepository userRepository;
+
+  @Autowired
+  public void setStudentRepository(StudentRepository studentRepository) {
+    this.studentRepository = studentRepository;
+  }
+
+  @Autowired
+  public void setUserRepository(UserRepository userRepository) {
+    this.userRepository = userRepository;
+  }
+
+  @Override
+  public void checkStudentFields(List<Student> students) {
```

Comment:
i fill like method name must be like fillNotUpdatedFields or smth like that. No validation logic here